app/lib/activity_stream_reader.rb

```diff
@@ -26,62 +36,59 @@ def process_page(page_url)
     process_page(previous_page_link(page)) if previous_page_link(page)
   end
 
+  def process_item(item)
+    dependent_uri = parse_item_identifier(item)
+    dependent_objects = dependent_objects_based_on(dependent_uri)
+    return false unless dependent_objects
+    add_to_parent_objects_for_update_set(dependent_objects)
+    @tally += 1
```

Comment:
It may be that @tally is off since it might be more than one parent.  

It's right if object_count is the number items with one or more document matches, but is seems like object_count might mean the total number of documents that were updated.

If it's the number of documents that were updated, it might be possible to get rid of tally and just use parent_objects_for_update.count for the log.object_count in process_activity_stream()